coco/request_forwarder.py

```diff
@@ -83,11 +119,14 @@ def add_endpoint(self, name, endpoint):
                 raise_for_status=False,
                 timeout=aiohttp.ClientTimeout(1),
             ) as response:
+                self.counter_succ[endpoint].labels(host=host_label, port=port).inc()
```

Comment:
If an exception is raised afterwards, both the success and the failure counter are incremented for this host.

Reply:
...but this is probably obsolete. use `response.status` for a label (name could be `status`). In case of the exception we could use `status=0` if you don't have a better idea

Reply:
Where could it raise an exception? if `session.request` fails, won't we not enter the block? In any case, I did what you suggested and moved to just one counter.

Reply:
`response.text` can raise a `LookupError`: https://docs.aiohttp.org/en/stable/client_reference.html#aiohttp.ClientResponse.text